bundler.d/aws.rb

```diff
@@ -0,0 +1,3 @@
+group :aws do
+  gem 'fog-aws'
```

Comment:
Adding its own bundler group means you'll be able to --exclude=fog (and others) for a minimal and faster install.  Our packages are split into one per bundler group, so you'll be able to install foreman-aws (or should that be "ec2"?) also.